sqlglot/parser.py

```diff
@@ -2974,7 +2970,7 @@ def _parse_unnest(self, with_alias: bool = True) -> t.Optional[exp.Unnest]:
 
     def _parse_derived_table_values(self) -> t.Optional[exp.Values]:
         is_derived = self._match_pair(TokenType.L_PAREN, TokenType.VALUES)
-        if not is_derived and not self._match(TokenType.VALUES):
+        if not is_derived and not self._match_text_seq("VALUES"):
```

Comment:
why text and not a token?

Reply:
hm, if bq and redshift have values statements, shouldn't they have the token?

Reply:
The motivation for getting rid of the token was that we don't wanna hit the `if match(VALUES)` branch in the `SELECT` parser, because that path is reached very often and we don't want that for BQ / Redshift. If we don't pop them off then we'll probably need more shenanigans with flags etc, like `if match(VALUES) and SUPPORTS_VALUES_CLAUSE: ...`. The insert parser would need to be updated either way I think.

Reply:
ok sounds good